src/containers/UserEdit.jsx

```diff
@@ -374,26 +374,33 @@ export class UserEditBase extends React.Component {
             userId === currentUser.currentUser.id &&
             !fieldsNeeded && (
               <div className={css(styles.container)}>
-                <Button onClick={this.handleClick} variant="outlined">
-                  Change password
-                </Button>
+                {window.PASSPORT_STRATEGY !== "auth0" && (
+                  <Button onClick={this.handleClick} variant="outlined">
+                    Change password
+                  </Button>
+                )}
               </div>
             )}
           <div className={css(styles.buttons)}>
-            <Form.Submit
-              as={GSSubmitButton}
-              className={css(styles.submit)}
-              label={saveLabel || "Save"}
-            />
-            {!authType && onCancel && !fieldsNeeded && (
-              <Button
-                className={css(styles.cancel)}
-                variant="outlined"
-                onClick={onCancel}
-              >
-                Cancel
-              </Button>
+            {window.PASSPORT_STRATEGY !== "auth0" && (
+              <Form.Submit
+                as={GSSubmitButton}
+                className={css(styles.submit)}
+                label={saveLabel || "Save"}
+              />
             )}
+            {!authType &&
+              onCancel &&
+              !fieldsNeeded &&
+              window.PASSPORT_STRATEGY !== "auth0" && (
+                <Button
+                  className={css(styles.cancel)}
+                  variant="outlined"
+                  onClick={onCancel}
+                >
+                  Cancel
+                </Button>
+              )}
```

Comment:
If the `Save` and `Cancel` buttons are removed, then users won't be able to update their first name, last name, texting alias, or cell phone number. Instead of removing the `Save` and `Cancel` buttons, can we just disable the ability to edit the Email field if Auth0 is enabled?

Reply:
We have changed this code so that it reflects the changes you requested. :)